sdcm/tester.py

```diff
@@ -338,7 +338,6 @@ class ClusterTester(db_stats.TestStatsMixin, unittest.TestCase):
     monitors: BaseMonitorSet = None
     loaders: Union[BaseLoaderSet, LoaderSetAWS, LoaderSetGCE] = None
     db_cluster: Union[BaseCluster, BaseScyllaCluster] = None
-    argus_heartbeat_stop_signal = threading.Event()
```

Comment:
I think this need to be clear from some unittests as well..

Reply:
removed